stripe/build.gradle

```diff
@@ -22,7 +22,7 @@ android {
     compileSdkVersion 25
     buildToolsVersion '25.0.0'
     defaultConfig {
-        minSdkVersion 9
```

Comment:
Increase required in order to do animation on our own. This will necessitate a major version bump, since Gingerbread phones (1.0% of the market) will no longer be supported.  For comparison: [Facebook's sdk requires a min of 15 at the time of writing this sentence](https://developers.facebook.com/docs/android/)